src/retreival_chain.py

```diff
@@ -17,6 +20,7 @@
 from conversational_chain.graph import RAGGraphWithMemory
 from conversational_chain.memory import ChatHistoryMemory
 from reactome.metadata_info import descriptions_info, field_info
+from util.embedding_environment import EmbeddingEnvironment
```

Comment:
Don't need this import - `embeddings_directory` is provided for `create_retrieval_chain()` below.